assets/webpack/webpack.dev.js

```diff
@@ -16,7 +16,7 @@ module.exports = merge(common, {
   devtool: "eval-source-map",
 
   devServer: {
-    port: process.env.PORT || 3001,
+    port: process.env.PORT || 3002,
     hot: true,
```

Comment:
Can we also add `disableHostCheck: true` here to work around cross origin errors on dev servers?